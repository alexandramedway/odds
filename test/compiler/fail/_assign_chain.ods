do x = y = 5
