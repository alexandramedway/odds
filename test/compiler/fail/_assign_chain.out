Analyzer Exception: Invalid attempt to chain assigns
